surch/organization.py

```diff
@@ -19,199 +19,200 @@
 
 import requests
 
-from . import logger, repo, utils, constants
-
-REPO_DETAILS_API_URL = \
-    'https://api.github.com/{0}/{1}/repos?type={2}&per_page={3}&page={4}'
-ORG_DETAILS_API_URL = 'https://api.github.com/{0}/{1}'
-
-
-lgr = logger.init()
+from . import repo, utils, constants
 
 
 class Organization(object):
     def __init__(
             self,
             search_list,
             organization,
-            git_user,
-            git_password,
+            verbose=False,
+            git_user=None,
+            results_dir=None,
+            git_password=None,
             print_result=False,
-            organization_flag=True,
             repos_to_skip=None,
             repos_to_check=None,
+            is_organization=True,
             consolidate_log=False,
-            cloned_repos_path=constants.CLONED_REPOS_PATH,
-            results_dir=constants.RESULTS_PATH,
-            verbose=False,
+            cloned_repos_dir=None,
             remove_cloned_dir=False,
             **kwargs):
-        """Surch instance define var from CLI or config file
+        """Surch org instance define var from CLI or config file
         """
+        utils.check_if_cmd_exists_else_exit('git')
+
+        self.logger = utils.logger
```

Comment:
Eventually I believe we should strive to something more like this:
try:
  MakeSureGitIsInstalled()
  MakeSureSureToSkipReposOr...()
except:
 print 'error'
sys.exit(1)
  sys.exit(1)